paging/paging-common/api/current.txt

```diff
@@ -134,7 +134,7 @@ package androidx.paging {
   }
 
   public enum LoadType {
-    method public static androidx.paging.LoadType valueOf(String value) throws java.lang.IllegalArgumentException, java.lang.NullPointerException;
+    method public static androidx.paging.LoadType valueOf(String name) throws java.lang.IllegalArgumentException, java.lang.NullPointerException;
```

Comment:
This API change (along with the others) seems to just be a reversion of https://github.com/androidx/androidx/commit/110d3a7c9775c5123a891145bb2ee31f2ab9ab29 from a couple of days ago. I was prompted to run `./gradlew updateApi` which resulted in these changes.

Reply:
Any chance you can try to rebase and re-run? GitHub got out of sync for metalava build (which does our api tracking) over night, but should be fixed now.

Reply:
Rebased! Thanks – after running `./gradlew updateApi` again, it undid all the supposed API changes.